api/views/session_views.py

```diff
@@ -264,3 +267,36 @@ def delete(self, request, code):
             return Response({"message": f"{str(e)}"}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 
 
+class LectureSummaryView(APIView):
+    permission_classes = [IsRecordingOwner]
+
+    def post(self, request):
```

Comment:
Can you please add a serializer and the Swagger decorator; we should be able to invoke the API from Swagger UI.